examples/skysync-cli-example-create-job-01.ps1

```diff
@@ -0,0 +1,77 @@
+<#
+.DESCRIPTION
+    Sample script in PowerShell to showcase the use of the CLI to generate and manage SkySync jobs. Example scenario:
+        * Create an NFS connection
+        * Create a Box connection
+        * Create a SkySync job that utilizes these connections
+        * Start the job
+
+.REFERENCE
+    https://skysync.atlassian.net/wiki/spaces/SSP/pages/15302683/SkySync+Command-line+Interface
+
+.NOTES
+    Setup/Config:
+        1. Create file 'skysync-cli.json' and place one directory above cli project/script that is being executed and add the following JSON block to the file:
+
+            {
+	        "output-json": true,
+	        "server": "http://localhost:9090/",
+	        "username": "***********",
+	        "password": "***********"
+            } 
+        2. Configure connection auth files
+        3. Update source and destination paths
+                Pathing Usage:
+                    a. Path Format Example for My Comp (FS): '/C/TestFiles/C7546397471/L0/B1'
+                    b. Path Format Example for Box: '/TEST/CLI_TEST_01'
+                    c. Path Format Example for NFS: '/C7546397471/L0/B1'  -start folder under configured Connection Share
+    Execution:
+        Run script from examples dir (cd .\~\skysync-cli\examples\skysync-cli-sample-script.ps1)
+#>
+
+$ErrorActionPreference = 'Stop'
+
+#set variables for paths
+$sourcePath = '/TestFolder1'
```

Comment:
can you update the variable name or add a note so that it's more clear that this is for  NFS

Reply:
done